docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Author [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
+```
+</center>
+
+## Key features
+
+* Similar experience when authoring [REST](api_gateway.md){target="_blank"} and [GraphQL APIs](appsync.md){target="_blank"}
```

Comment:
I think we should remove this line or explain how it's similar, lets not expect prior knowledge of PT features

Reply:
You have a good point, specially this being the first key feature. So I'm moved it to the last bullet point.